kodilla-intro/src/main/java/UserApplication.java

```diff
@@ -0,0 +1,39 @@
+public class UserApplication {
+    double aveAge;
```

Comment:
to sa teraz pola w klasie UserApplication - po co? To moga byc zmienne (czyli zadeklarowane w metodzie)